### DISCUSSION TITLE
Difference between "Downloaded" and "Available right now"

### DISCUSSION THREAD

**Discussion by SturmB** [xbapps/xbvr#1629] [Q&A]
I finally got everything set up, as far as I can tell. However, XBVR's web UI only shows 4 of 122 videos as "Available right now". What does that mean? I haven't tried DeoVR yet, but according to [this Q&A (with an insufficient answer)](https://discuss.eroscripts.com/t/xbvr-available-right-now-versus-downloaded/60528), any videos not in the "Available right now" tab won't appear in DeoVR. Is this true? And what must be done to make "Downloaded" videos appear in "Available right now"?

![image](https://github.com/xbapps/xbvr/assets/6216224/631ba566-9f55-45b5-ada9-ab15f7a72f7e)

**Comment by theRealKLH** [comment]
showing in "Downloaded" but not in "Available" means xbvr does not see the volume that your videos are on. Most likely reason is you have your videos on external drives that may not be connected or the drive letters/paths have changed?

**Reply by SturmB** [reply]
That would make sense, except that all 122 videos are on the same volume, on internal HDDs, and even in the same path tree. In addition, they all have the same owner/group and permissions.

I'm running a rescan of the entire folder, hoping that might help it to realize that all of the videos are always available.

**Comment by SturmB** [comment]
I thought I might try starting over by removing the folder from Storage and letting it re-scan everything. Well, that removed everything from the "Available Now" section, but it's still showing all my files in the "Downloaded" section, as well as under "Files". I don't want to delete the files, so how can I get XBVR to start over with scanning my files in the hopes that it will realize that they're all on the same local volume and, thus, should all be available?

**Comment by swapsingh200** [comment]
Was facing the same problem, a solution that worked for me:
1. I didn't change any xbvr settings, and used file explorer to rename my directory with all vids (e.g. /pathname/VR) to something temp like (/pathname/tempVR).
2. On xbvr, I rescanned the /pathname/VR directory (options -> storage -> rescan)
3. Back on the file explorer, I changed the directory name back to original ( /pathname/VR)
4. Finally on xbvr, I rescanned  /pathname/VR again.
